airbyte-integrations/connectors/source-codefresh/manifest.yaml

```diff
@@ -0,0 +1,10878 @@
+version: 5.10.2
+
+type: DeclarativeSource
+
+description: >-
+  This connector integrates Codefresh with Airbyte, enabling seamless data
+  synchronization for CI/CD analytics and pipeline monitoring. It provides
+  streams like agents, builds, audit, analytics etc.
+
+check:
+  type: CheckStream
+  stream_names:
+    - accounts
+
+definitions:
+  streams:
+    accounts:
+      type: DeclarativeStream
+      name: accounts
```

Comment:
This is `account_users`. There is a `accounts` endpoint where you can use as Parent Stream to a lot of streams. This way you can remove the `account_id` from the config.

Reply:
@marcosmarxm are you talking about the endpoint which i shared in the screenshot below? if so then Its a admin only endpoint . I tried that already. It gives me insufficient permission error
<img width="1706" alt="Screenshot 2024-10-08 at 8 50 08 AM" src="https://github.com/user-attachments/assets/e1603fc0-0864-4889-80fe-8259c3f12c07">

Reply:
You need to recreate the API key you're using.
<img width="560" alt="image" src="https://github.com/user-attachments/assets/7ee898f5-a1d8-4afa-8754-7d40b079a986">

Reply:
why? I had given it full access. @marcosmarxm 
<img width="1051" alt="Screenshot 2024-10-17 at 5 34 53 AM" src="https://github.com/user-attachments/assets/a24b8e9a-dfd5-484a-9f82-851738081bba">